tethysext/atcore/controllers/resource_workflows/results_views/report_workflow_results_view.py

```diff
@@ -0,0 +1,221 @@
+"""
+********************************************************************************
+* Name: report_workflow_results_view.py
+* Author: nswain, htran, msouffront
+* Created On: October 14, 2020
+* Copyright: (c) Aquaveo 2020
+********************************************************************************
+"""
+import logging
+from tethysext.atcore.models.resource_workflow_results import ReportWorkflowResult
+from tethysext.atcore.controllers.resource_workflows.map_workflows import MapWorkflowView
+from tethysext.atcore.controllers.resource_workflows.workflow_results_view import WorkflowResultsView
+from tethysext.atcore.models.resource_workflow_results import DatasetWorkflowResult, PlotWorkflowResult,\
+    SpatialWorkflowResult
+from tethysext.atcore.controllers.utilities import get_plot_object_from_result, get_tabular_data_for_previous_steps
+
+from tethys_sdk.gizmos import DataTableView
+from collections import OrderedDict
+
+
+log = logging.getLogger(__name__)
+
+
+class ReportWorkflowResultsView(MapWorkflowView, WorkflowResultsView):
+    """
+    Report Result View controller.
+    """
+    template_name = 'atcore/resource_workflows/report_workflow_results_view.html'
+    valid_result_classes = [ReportWorkflowResult]
+
+    previous_steps_selectable = True
+
+    def process_step_options(self, request, session, context, resource, current_step, previous_step, next_step):
+        """
+        Hook for processing step options (i.e.: modify map or context based on step options).
+
+        Args:
+            request(HttpRequest): The request.
+            session(sqlalchemy.orm.Session): Session bound to the steps.
+            context(dict): Context object for the map view template.
+            resource(Resource): the resource for this request.
+            current_step(ResourceWorkflowStep): The current step to be rendered.
+            previous_step(ResourceWorkflowStep): The previous step.
+            next_step(ResourceWorkflowStep): The next step.
+        """
+        # Turn off feature selection on model layers
+        map_view = context['map_view']
+        self.set_feature_selection(map_view=map_view, enabled=False)
+
+        # Can run workflows if not readonly
+        can_run_workflows = not self.is_read_only(request, current_step)
+
+        # get tabular data if any
+        tabular_data = get_tabular_data_for_previous_steps(
```

Comment:
Is tabular_data needed for the report result view? Seems like it is here to satisfy a need in the template that isn't needed.

Reply:
Got it. Thanks for the clarification.